docs/user-guide/security-features.md

```diff
@@ -1,152 +1,152 @@
 
-This Security Feature has two processes:
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/scan-report-on-build-history.jpg)
 
-1. Scanning
-2. Policy
+ The scan report provides a comprehensive overview of any vulnerabilities present in the image. This allows you to identify and address potential security risks effectively. By leveraging this feature, you can ensure that your containerized applications are safeguarded against known vulnerabilities.
 
-## Scanning
+Devtron's Security Feature consists of two primary components:
 
-This process starts executing after the successful execution of the CI pipeline and before the deployment\(CD\) process starts.
+1. **Security Scans**
+2. **Security Policies**
 
-It scans your code to find any potential threat and shows you the list of vulnerabilities as an output of the CI pipeline if it finds any.
+## Security Scans
 
-We will discuss later how you will see the list of your found vulnerabilities.
+Devtron's security scans provide comprehensive scan reports for all applications that have undergone vulnerability scanning. These reports offer a detailed overview of the security status of each scanned application.
 
-## Policy
+These comprehensive scan reports provide valuable insights, including information about identified vulnerabilities, their severity levels, and any corresponding Common Vulnerabilities and Exposures (CVE) entries.
 
-Vulnerabilities have different levels like Critical, Moderate, and Low. Users can define policy according to the level of vulnerability. Users can also block the vulnerability or allow\(whitelist\) the vulnerability for their code.
+## Security Policies
 
-If any vulnerability is found which is blocked by the user, then it will not deploy the application. And if it finds any vulnerability which is whitelisted by the user, then the build image can be deployed.
+Devtron's Security Policies feature allows users to define policies based on the severity levels of vulnerabilities, which include **Critical**, **Moderate**, and **Low**. Users have the flexibility to set policies that either block the deployment of container images with vulnerabilities or allow their deployment.
 
-The user gets informed in both cases if it finds any vulnerability or doesn't find any.
+With this feature, users can specify their desired actions for each severity level. For example, they can choose to block any container image with Critical vulnerabilities, while allowing container images with Moderate or Low vulnerabilities to be deployed.
```

Comment:
With this feature, users can specify their desired actions for each severity level. For example, they can choose to block any container image with `Critical` vulnerabilities, while allowing container images with `Moderate` or `Low` vulnerabilities to be deployed.